i2c_manager/Kconfig

```diff
@@ -0,0 +1,116 @@
+menu "I2C Port Settings"
+    depends on LV_I2C && !HAVE_I2C_MANAGER
+
+	menu "I2C Port 0"
+
+		config I2C_MANAGER_0_ENABLED
+			bool "Enable I2C port 0"
+
+		if I2C_MANAGER_0_ENABLED
+			config I2C_MANAGER_0_SDA
+				int "SDA (GPIO pin)"
+				range 0 39 if IDF_TARGET_ESP32
+				range 0 43 if IDF_TARGET_ESP32S2
+				range 0 21 if IDF_TARGET_ESP32C3
+			config I2C_MANAGER_0_SCL
+				int "SCL (GPIO pin)"
+				range 0 39 if IDF_TARGET_ESP32
+				range 0 43 if IDF_TARGET_ESP32S2
+				range 0 21 if IDF_TARGET_ESP32C3
+			config I2C_MANAGER_0_FREQ_HZ
+				int "Frequency (Hz)"
+				default 1000000
```

Comment:
According to [this](https://docs.espressif.com/projects/esp-idf/en/v4.1.1/api-reference/peripherals/i2c.html#_CPPv4N12i2c_config_t9clk_speedE) I2C clock speed can't be higher than 1_000_000, so i think setting this to 100_000 is safer, what do you think?

Reply:
Default now 400 kHz, not 1 MHz.